### DISCUSSION TITLE
Release Schedule

### DISCUSSION THREAD

**Discussion by jSantos-Eletric** [aminalaee/sqladmin#566] [General]
Hey, I'm looking forward to start using some changes that were made last month, like https://github.com/aminalaee/sqladmin/pull/547, do you have any plans on when are you making that release?

**Comment by aminalaee** [comment]
Hey, I will prepare that tomorrow or the day after.

**Reply by aminalaee** [reply]
https://github.com/aminalaee/sqladmin/pull/567
